packages/webapp/.env

```diff
@@ -7,5 +7,5 @@ NEXT_PUBLIC_WEBAPP_URL=/
 NEXT_PUBLIC_AUTH_URL=http://sso.local.com
 NEXT_PUBLIC_HEIMDALL_URL=http://sso.local.com
 #NEXT_PUBLIC_AUTH_URL=https://sso.daily.dev
-NEXT_PUBLIC_AUTH_URL=http://sso.apps.local.app.garden
-NEXT_PUBLIC_HEIMDALL_URL=http://heimdall.apps.local.app.garden
+
+NEXT_PUBLIC_ONESIGNAL_APP_ID=04cddacf-5920-47a6-b567-9ae334d26681
```

Comment:
We need to set Vercel as well with the production setting

Reply:
Alright. Let me add this now for production setting.

Reply:
We might need another OneSignal App for our staging/preview environment as the URL is different from the webapp production URL.

Reply:
I'll create one for our preview deployment and send you so you can set it on Vercel

Reply:
Thank you! 🚢

Reply:
Thinking about it, we can't set a different one for the preview as it works against our production backend.
My suggestion would be to temporarily change the url of production for the testing purposes and then revert. How does it sound?

Reply:
Sounds good to me. This will be a one time testing anyways as there will be not much things to change once setup is successful.